mteb/tasks/BitextMining/multilingual/NTREXBitextMining.py

```diff
@@ -55,12 +161,12 @@ class NTREXBitextMining(AbsTaskBitextMining, CrosslingualTask):
     metadata = TaskMetadata(
         name="NTREXBitextMining",
         dataset={
-            "path": "xianf/NTREX",
-            "revision": "6047adbf00ceeab9b543bd0bd779b99866deddc0",
+            "path": "davidstap/NTREX",
+            "revision": "fd20d54141b6da952d5c68a2989472892489da0f",
             "trust_remote_code": True,
         },
         description="NTREX is a News Test References for MT Evaluation from English into a total of 128 target languages.",
```

Comment:
Should be changed in imagine

Reply:
I'm not sure what you mean?

Reply:
Sorry again for the short comments, seems like I might have been doing a lot of review that day.

The description write from English to 128 languages, which seems to imply only one direction -- English to 128 languages. While I believe this is the description of the NTREX data, the description should be of the given current task (including any modification we might have made).

Reply:
I see, will update the description!